client/src/components/Related/ProductCard.jsx

```diff
@@ -75,7 +74,7 @@ const ProductCard = (props) => {
         <StarsTile stars ={starAverage} />
       </div>
       <button onClick={openComparisonModal}>show comparison window</button>
```

Comment:
maybe take out the window word, so button reads: show comparison. Sounds nicer.

Reply:
fixed